experiment/autobumper/bumper/bumper.go

```diff
@@ -228,6 +231,10 @@ func validateOptions(o *Options) error {
 	if !o.SkipPullRequest && o.HeadBranchName == "" {
 		o.HeadBranchName = defaultHeadBranchName
 	}
+	// TODO: could auto-detect the target repo's default base branch
```

Comment:
/retitle [wip] autobumper: support base branches other than master
Discussed offline, I'll address this TODO before removing the wip label